src/services/tx/sponsoredCall.ts

```diff
@@ -0,0 +1,31 @@
+import { SAFE_GELATO_RELAY_SERVICE_URL } from '@/config/constants'
+import { type SafeTransactionData } from '@safe-global/safe-core-sdk-types'
+
+// TODO: import type from relay-service
+export type SponsoredCallPayload = {
+  chainId: string
+  to: string
+  data: SafeTransactionData['data'] | undefined
```

Comment:
`data` cannot be `undefined`.

```suggestion
  data: SafeTransactionData['data']
```